packages/cli/src/commands/link/android/patches/applyPatch.js

```diff
@@ -8,8 +8,13 @@
  */
 
 import fs from 'fs';
+import logger from '../../../../tools/logger';
```

Comment:
This is getting really ugly. @thymikee Do you think we should introduce `absolute` paths to the project?

Reply:
Sure